test/tools/libtesteth/ImportTest.cpp

```diff
@@ -226,8 +226,7 @@ std::tuple<eth::State, ImportTest::ExecOutput, eth::ChangeLog> ImportTest::execu
 			st.setShowMnemonics();
 			st.setOptions(Options::get().jsontraceOptions);
 			out = initialState.execute(_env, *se.get(), _tr, Permanence::Committed, st.onOp());
-			cout << st.json();
-			cout << "{\"stateRoot\": \"" << initialState.rootHash().hex() << "\"}";
+			cout << st.json() + "{\"stateRoot\": \"" + initialState.rootHash().hex() + "\"}";
```

Comment:
Is this change necessary?

Reply:
no. if we disabled travis out thread it should be fine without this.